src/Shared/CloudRepublic.BenchMark.Domain/Enums/Runtime.cs

```diff
@@ -2,8 +2,10 @@ namespace CloudRepublic.BenchMark.Domain.Enums
 {
     public enum Runtime
```

Comment:
Deze hernoemen naar Language, Zodat we later de echte Runtime kunnen toevoegen voor versie 2/3 van functions.